helpers/core.js

```diff
@@ -42,19 +42,33 @@ var helpers = {
 			options = args.pop(),
 			argsLen = args.length,
 			argExprs = options.exprData.argExprs,
+			hashExprs = options.exprData.hashExprs,
 			resolved = resolve(items),
 			asVariable,
+			hashOptions,
 			aliases,
 			key;
 
 		if (argsLen === 2 || (argsLen === 3 && argExprs[1].key === 'as')) {
+			//!steal-remove-start
+			dev.warn('Using the `as` keyword is deprecated in favor of hash expressions. https://canjs.com/doc/can-stache.helpers.each.html');
```

Comment:
Can we make this warning give the correct new form?

Like `{{#each items as item}}` should now be `{{#each items item=value}}`.